src/unistd.rs

```diff
@@ -356,6 +375,13 @@ pub fn getcwd() -> Result<PathBuf> {
     }
 }
 
+// According to the POSIX specification, -1 is used to indicate that
```

Comment:
This comment doesn't make sense here. Probably want to put it back in the function.